__tests__/triggers/user.ts

```diff
@@ -41,38 +47,46 @@ describe('user', () => {
       await saveFixtures(con, User, usersFixture);
       await saveFixtures(con, User, badUsersFixture);
       await saveFixtures(con, Source, sourcesFixture);
-      await saveFixtures(con, Post, postsFixture);
+      await saveFixtures(con, ArticlePost, [
+        {
+          ...postsFixture[0],
+          id: 'pvr1',
+          shortId: 'pvr1',
+          url: 'http://pvr1.com',
+          canonicalUrl: 'http://pvr1c.com',
+        },
+      ]);
       await saveFixtures(con, Comment, [
         {
-          id: 'c1',
-          postId: 'p1',
+          id: 'cvr1',
+          postId: 'pvr1',
```

Comment:
We should probably strive to avoid using fixtures directly and make unique ids across different files to avoid this kind of issues (we do it in recent features either way).